save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/SpringUtils.kt

```diff
@@ -49,3 +50,24 @@ fun <K> Storage<K>.overwrite(key: K, content: Part): Mono<Long> = content.conten
  */
 fun <K> Storage<K>.overwrite(key: K, contentMono: Mono<Part>): Mono<Long> = contentMono
     .flatMap { overwrite(key, it) }
+
+/**
+ * @return [WebClient] with enabled Huawei proxy
+ */
+@Suppress(
+    "COMPLEX_EXPRESSION",
+    "MAGIC_NUMBER", "MagicNumber",
+)
+fun WebClient.enableHuaweiProxy(): WebClient = this
+    .mutate()
+    .clientConnector(
+        org.springframework.http.client.reactive.ReactorClientHttpConnector(
+            reactor.netty.http.client.HttpClient.create()
```

Comment:
Does `reactor-netty` respect `-Dhttp.proxyHost` and `-Dhttp.proxyPort`? Anyway, I think proxy configuration should be set externally